supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -20,11 +20,19 @@ For more information about <topic>, see xref:<link>[<link_text>].
 
 Follow these guidelines when linking externally:
 
-* Always include link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
+* Always include accurate link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
 * Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. For information about using addresses in examples, see the _IBM Style_ guide.
-* By default, links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
-* Avoid links to external sites when possible. External links can change or be unreliable.
-* Avoid deep links (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
+* By default, ensure that links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
+* When possible, link to a top-level page and avoid deep links to a specific page or image.
+Linking to a top-level page helps minimize future link maintenance and can avoid inadvertent bypassing of a site's legal information.
+* When feasible, avoid links to external sites, which are sites other than those owned and operated by Red Hat or IBM.
```

Comment:
```suggestion
* Whenever possible, avoid links to external sites, which are sites other than those owned and operated by Red Hat or IBM.
```
Non-native speakers might have a hard time with "feasible".

Reply:
You're right: "possible" is better than "feasibel"